database.go

```diff
@@ -86,7 +86,7 @@ func createDatabase() {
 		email TEXT NOT NULL UNIQUE,
 		username TEXT NOT NULL UNIQUE,
 		password TEXT NOT NULL,
-		user_type TEXT NOT NULL
+		is_root INTEGER NOT NULL
```

Comment:
Ok, I did this because the Stackoverflow says as follows:

"SQLite does not have a separate Boolean storage class. Instead, Boolean values are stored as integers 0 (false) and 1 (true)."

Even says:

"CREATE TABLE foo(mycolumn BOOLEAN NOT NULL CHECK (mycolumn IN (0,1)));

and some that will fail:

sqlite> INSERT INTO foo VALUES("true");
Error: constraint failed"

Reply:
`BOOLEAN` values are of the `NUMERIC` affinity. That's fine. Bear in mind that we will move to postgresql in production which does have booleans.

Reply:
Undesrtood